### ISSUE TITLE
TODOs for the experiment

### ISSUE THREAD

**Issue by titipata** [titipata/detecting-scientific-claim#13]
List of improvements and things to do as @csbhagav suggested

- [x] Create a snippet to prepare the dataset for the annotation
- [x] Loss for the CRF layer is too high >> check the loss function
- [x] Adding `"evaluate_on_test": true,` to the experiment JSON file
- [x] Trying `trainable = true` for Glove embeddings
- [x] Adding Learning Rate Schedule to the experiment 
- [x] Sequence prediction output for CRF models
- [x] Alternate training between discourse and claim

**Comment by titipata** [comment]
See [here](https://github.com/allenai/allennlp/blob/09c2cc5616a8aec897f8fe06fda13179955bec69/allennlp/tests/fixtures/esim/experiment.json#L87-L91) for the Learning rate scheduler example.

**Comment by titipata** [comment]
Added TODO: adding the sequence prediction output for Conditional Random Field case.
